src/test/java/gov/usgs/wma/waterdata/groundwater/RdbWriterTest.java

```diff
@@ -57,7 +53,7 @@ DiscreteGroundWater makeDgw() {
 		dgw.dateTimeAccuracyCode = "D"; // [D]day or [M]minute
 		dgw.timezoneCode = "UTC";
 		dgw.timeMeasuredUtc = "01-MAY-2007 12:00:00"; // UTC
-		dgw.approvalStatusCode = "T"; // T or R
```

Comment:
I'm not sure where the "T" or "R" concept came from, but it was not being passed thru to the rdb file, so it didn't really matter.